components/serializer.rst

```diff
@@ -413,6 +413,80 @@ As for groups, attributes can be selected during both the serialization and dese
 Ignoring Attributes
 -------------------
 
+Set `false` by default, you can ignore attribute by two methods.
+
+* `Option 1: Using annotation @Ignore`_
+* `Option 2: Using the context`_
+
+Option 1: Using annotation @Ignore
+----------------------------------
+
+.. configuration-block::
+
+    .. code-block:: php-annotations
+
+        namespace Acme;
+
+        use Symfony\Component\Serializer\Annotation\Ignore;
+
+        class MyObj
+        {
+            public $foo;
+            /**
+             * @Ignore()
+             */
+            public $bar;
+        }
+
+    .. code-block:: yaml
+
+        Acme\MyObj:
+            attributes:
+                foo:
+                    groups:
+                        ignore: false
+                bar:
+                    groups:
+                        ignore: true
+
+    .. code-block:: xml
+
+        <?xml version="1.0" ?>
+        <serializer xmlns="http://symfony.com/schema/dic/serializer-mapping"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://symfony.com/schema/dic/serializer-mapping
+                https://symfony.com/schema/dic/serializer-mapping/serializer-mapping-1.0.xsd"
+        >
+            <class name="Acme\MyObj">
+                <attribute name="foo">
+                    <ignore>false</ignore>
+                </attribute>
+
+                <attribute name="bar">
+                    <ignore>true</ignore>
+                </attribute>
+            </class>
+        </serializer>
+
+You are now able to ignore specifics attributes during serialization::
+
+    use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
+    use Symfony\Component\Serializer\Serializer;
+
+    $obj = new MyObj();
+    $obj->foo = 'foo';
+    $obj->bar = 'bar';
+
+    $normalizer = new ObjectNormalizer($classMetadataFactory);
+    $serializer = new Serializer([$normalizer]);
+
+    $data = $serializer->normalize($obj);
+    // $data = ['foo' => 'foo'];
+
+
+Option 2: Using the context
+---------------------------------
+
 As an option, there's a way to ignore attributes from the origin object.
```

Comment:
This phrase needs some update:

```
As an option, there's a way to ignore attributes from the origin object.
```

Because now this is not the only way of achieving this and we already explained another way of doing it before.

Reply:
Wdyt about new sentence ?